test/unit/spec/narrative_core/kbaseNarrativeStagingDataTab-spec.js

```diff
@@ -47,9 +52,23 @@ define(['jquery', 'kbaseNarrativeStagingDataTab', 'base/js/namespace'], (
                         mtime: 1532738637555,
                         size: 49233,
                         source: 'KBase upload',
+                        isFolder: false,
                     },
                 ]),
             });
+            const mappings = {
+                mappings: [null, null],
+            };
+            jasmine.Ajax.stubRequest(
```

Comment:
This wasn't being mocked, so was actually trying to call out, which was a big problem.

Reply:
😖